Solutions/Amazon Web Services NetworkFirewall/Data Connectors/AWSNetworkFirewallLogs_CCP/AWSNetworkFirewallLog_ConnectorDefinition.json

```diff
@@ -0,0 +1,191 @@
+			"graphQueriesTableName": "AWSNetworkFirewall_FlowLog_CL",
+			"graphQueries": [
+				{
+					"metricName": "Total Flow events received",
+					"legend": "Amazon Web Services NetworkFirewall FlowLog",
+					"baseQuery": "{{graphQueriesTableName}}"
+				},
+				{
+					"metricName": "Total Alerts received",
+					"legend": "Amazon Web Services NetworkFirewall AlertLog",
+					"baseQuery": "AWSNetworkFirewall_AlertLog_CL"
+				},
+				{
+					"metricName": "Total Alerts received",
+					"legend": "Amazon Web Services NetworkFirewall TLSLog",
+					"baseQuery": "AWSNetworkFirewall_TlsLog_CL"
+				}
+			],
+			"sampleQueries": [
+				{
+					"description": "Get Sample of Flow logs",
+					"query": "{{graphQueriesTableName}}\n | take 10"
+				},
+				{
+					"description": "Get Sample of Alert logs",
+					"query": "AWSNetworkFirewall_AlertLog_CL\n | take 10"
+				},
+				{
+					"description": "Get Sample of Tls logs",
+					"query": "AWSNetworkFirewall_TlsLog_CL\n | take 10"
+				}
+			],
+			"dataTypes": [
+				{
+					"name": "{{graphQueriesTableName}}",
+					"lastDataReceivedQuery": "{{graphQueriesTableName}}\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+				},
+				{
+					"name": "AWSNetworkFirewall_AlertLog_CL",
+					"lastDataReceivedQuery": "AWSNetworkFirewall_AlertLog_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+				},
+				{
+					"name": "AWSNetworkFirewall_TlsLog_CL",
+					"lastDataReceivedQuery": "AWSNetworkFirewall_TlsLog_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+				}
+			],
+			"connectivityCriteria": [
+				{
+					"type": "HasDataConnectors",
+					"value": null
+				}
+			],
+			"availability": {
+				"status": 1,
+				"isPreview": false
+			},
+			"permissions": {
+				"resourceProvider": [
+					{
+						"provider": "Microsoft.OperationalInsights/workspaces",
+						"permissionsDisplayText": "Read and Write permissions are required.",
+						"providerDisplayName": "Workspace",
+						"scope": "Workspace",
+						"requiredPermissions": {
+							"read": true,
+							"write": true,
+							"delete": true,
+							"action": false
+						}
+					}
+				]
+			},
+			"instructionSteps": [
+				{
+					"instructions": [
+						{
+							"type": "Markdown",
+							"parameters": {
+								"content": "#### 1. AWS CloudFormation Deployment \n To configure access on AWS, two templates need to be executed in AWS  \n You can find the OIDC Web Identity Provider, AWS Network Firewall Configuration Templates [Here](https://github.com/Azure/Azure-Sentinel/tree/Srinivas-AWS-Netwrok-Firewall/Solutions/Amazon%20Web%20Services%20Network%20Firewall/Data%20Connectors/CloudFormationTemplets) \n You can find the detailed steps for deploying the templates [Here](https://github.com/Azure/Azure-Sentinel/blob/Srinivas-AWS-Netwrok-Firewall/Solutions/Amazon%20Web%20Services%20Network%20Firewall/Data%20Connectors/readme.md)"
```

Comment:
https://github.com/Azure/Azure-Sentinel/tree/Srinivas-AWS-Netwrok-Firewall/Solutions/Amazon%20Web%20Services%20Network%20Firewall/Data%20Connectors/CloudFormationTemplets
Link not working